src/main/scala/Silicon.scala

```diff
@@ -44,10 +44,7 @@ object Silicon {
     s"${BuildInfo.projectVersion}${buildVersion.fold("")(v => s" ($v)")}"
 
   val copyright = "(c) Copyright ETH Zurich 2012 - 2019"
-  val z3ExeEnvironmentVariable = "Z3_EXE"
-  val z3MinVersion = Version("4.5.0")
-  val z3MaxVersion: Option[Version] = None // Some(Version("4.5.0")) /* X.Y.Z if that is the *last supported* version */
-  val dependencies = Seq(SilDefaultDependency("Z3", z3MinVersion.version, "https://github.com/Z3Prover/z3"))
+  val dependencies = Z3ProverStdIO.dependencies ++ Cvc5ProverStdIO.dependencies
```

Comment:
What does this do? Similarly to the comment in the README file: is it clear that both of these are not required dependencies?

Reply:
I moved the dependency definitions from Silicon.scala to the respective ProverStdIO files to avoid cluttering Silicon.scala with further prover-specific information.

I am not entirely sure how to clarify that only the Z3-dependency is needed by default and that the cvc5-dependency is only for when using the cvc5 prover. If you have any ideas, please let me know.

Reply:
If I'm not mistaken, these dependencies are only tracked this way so that Silver can report them, not sure if that feature has ever been used, though. I also don't think that the mechanism can differentiate between mandatory and optional dependencies, or dependencies that depend on command-line options.

@lfwa No need for further changes.